### DISCUSSION TITLE
Flaresolver 3.0.0

### DISCUSSION THREAD

**Discussion by Kazak-ops** [FlareSolverr/FlareSolverr#655] [Q&A]
Hi there, quick question, is this the build that helps with fixing the issue with flaresolverr not working lately? I've been using a beta docker build and wondering if i can switch back to this official one now

**Comment by ilike2burnthing** [comment]
Yes.

**Comment by ngosang** [comment]
Jackett v0.20.2495 + FlareSolverr 3.0.0 fix most issues as today
